src/targets/gpu/mlir.cpp

```diff
@@ -622,12 +669,21 @@ struct mlir_program
         mlir_pass_manager pm_back{mlirPassManagerCreate(ctx.get())};
         // 1st pipeline to call
         mlirMIGraphXAddHighLevelPipeline(pm_front.get());
-        mlirPassManagerRunOnOp(pm_front.get(), mlirModuleGetOperation(mmodule.get()));
+        if(mlirLogicalResultIsFailure(
```

Comment:
We should clear the log before running a pass.

Reply:
@pfultz2 Do youn know what the clang-tidy warning just above here is complaining about?